core/deliverservice/deliveryclient.go

```diff
@@ -290,16 +291,19 @@ func (d *deliverServiceImpl) newClient(chainID string, ledgerInfoProvider blocks
 	broadcastSetup := func(bd blocksprovider.BlocksDeliverer) error {
 		return requester.RequestBlocks(ledgerInfoProvider)
 	}
-	backoffPolicy := func(attemptNum int, elapsedTime time.Duration) (time.Duration, bool) {
+	backoffPolicy := func(attemptNum int, elapsedTime time.Duration, yieldLeadership bool) (time.Duration, bool) {
 		if elapsedTime >= reconnectTotalTimeThreshold {
-			return 0, false
+			if yieldLeadership {
+				return 0, false
+			}
+			logger.Warning("peer is a static leader, ignoring peer.deliveryclient.reconnectTotalTimeThreshold")
 		}
 		sleepIncrement := float64(time.Millisecond * 500)
 		attempt := float64(attemptNum)
 		return time.Duration(math.Min(math.Pow(2, attempt)*sleepIncrement, reconnectBackoffThreshold)), true
 	}
 	connProd := comm.NewConnectionProducer(d.conf.ConnFactory(chainID, d.connConfig.OrdererEndpointOverrides), d.connConfig.toEndpointCriteria())
-	bClient := NewBroadcastClient(connProd, d.conf.ABCFactory, broadcastSetup, backoffPolicy)
+	bClient := NewBroadcastClient(connProd, d.conf.ABCFactory, broadcastSetup, backoffPolicy, d.conf.IsStaticLeader)
```

Comment:
If we don't wire this, we should be able to encapsulate the desired behavior within the `backoffPolicy`.

In my mind, that means we have two policies - the one that existed before these changes and one for a static configuration that always returns `0, false`.